docs/develop/dotnet/cancellation.mdx

```diff
@@ -0,0 +1,72 @@
+---
+id: cancellation
+title: Cancellation - .NET SDK feature guide
+sidebar_label: Cancellation
+description: How to cancel a Workflow Execution and it's Activities using the .NET SDK.
+slug: /develop/dotnet/cancellation
+toc_max_heading_level: 4
+keywords:
+  - cancellation
+  - dotnet sdk
+  - overview
+tags:
+  - cancellation
+  - dotnet-sdk
+  - overview
+---
+
+This pages shows the following:
+
+- How to handle a Cancellation request within a Workflow.
+- How to set an Activity Heartbeat Timeout.
+- How to listen for and handle a Cancellation request within an Activity.
+- How to send a Cancellation request from a Temporal Client.
+
+## Handle Cancellation in Workflow {#handle-cancellation-in-workflow}
+
+**How to handle a Cancellation in a Workflow in .NET**
+
+<!-- CONTENT-->
+
+## Handle Cancellation in an Activity {#handle-cancellation-in-an-activity}
+
+**How to handle a Cancellation in an Activity in .NET**
+
+<!-- CONTENT-->
+
+## Request Cancellation {#request-cancellation}
+
+**How to request Cancellation of a Workflow and Activities in .NET**
+
+<!-- CONTENT-->
+
+## How to interrupt a Workflow Execution {#interrupt-a-workflow-execution}
```

Comment:
We are inconsistent with whether we put "How to" in the heading title (e.g. it's "Request Cancellation" not "How to request Cancellation")